Las/Utils/Authorization.cs

```diff
@@ -0,0 +1,194 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+using Lucidtech.Las.Core;
+
+namespace Lucidtech.Las.Utils
+{
+    public class AmazonAuthorization
+    {
+        /* General information */
+        private string Region { get; }
+        private string Service { get; }
+
+        /* Keys the customer receives from Lucidtech */
+        private string AwsApiKey { get; }
```

Comment:
I suggest just keeping a reference to a Credentials object and access the keys from Credentials object.

Reply:
Agree